AndroidTrainingModule/res/layout/relative_layout_activity.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+    
+    <TextView 
+        android:id="@+id/main_textView"
+        android:gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:textSize="30sp"
+        android:text="@string/hello_world"/>
+
+    <EditText
+        android:id="@+id/main_editText"
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/main_textView"
+        android:ems="10"
+        android:gravity="center"
+        android:hint="Enter new greeting.."
```

Comment:
You may create new field in `strings.xml` file then use it in setting the hint text.
In your strings.xml file: `<string name="any_name">Your Text</string>`
In your layout: `android:hint="@string/any_name"`